sklego/meta/_grouped_utils.py

```diff
@@ -0,0 +1,62 @@
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+from scipy.sparse import issparse
+from sklearn.utils import check_array
+from sklearn.utils.validation import _ensure_no_complex_data
+
+from sklego.common import as_list
+
+
+def split_groups_and_values(
```

Comment:
Is this a function that the user is supposed to touch? If not, might be better to implement them as `_split_groups_and_values`. 

Out of curiosity, do we use these functions in multiple places or just in the groupedtransformer? If it's the latter, is there a particular reason why you felt it better to have them in a separate file?

Reply:
Alright, will add the underscores.

I put them here because I think in a different PR it would be nice to use these for the `GroupedEstimator` as well (#390), do you prefer to keep them in this file for now?

Reply:
This is fine. Given the future plan I can certainly see the merits.